invenio_rdm_records/external_pids/providers.py

```diff
@@ -0,0 +1,28 @@
+
+
+class Provider:
```

Comment:
we have a base `provider` and for datacite too in [invenio-pidstore](https://github.com/inveniosoftware/invenio-pidstore/blob/master/invenio_pidstore/providers/datacite.py) this naming is confusing 😅. Shall we move them there/rename? whats the vision on this aspect?

Reply:
IRL: continue with this, having in mind the base provider of pidstore, to simplify things for now.